solidity/contracts/utility/TokenHolder.sol

```diff
@@ -33,11 +33,10 @@ contract TokenHolder is ITokenHolder, Owned, Utils {
         IERC20 _token,
         address _to,
         uint256 _amount
-    ) public virtual override ownerOnly validAddress(address(_token)) validAddress(_to) notThis(_to) {
+    ) public virtual override ownerOnly validAddress(address(_token)) validAddress(_to) {
```

Comment:
Shouldn't we check `validExternalAddress(_to)`? I mean, if anything, then `_to` is likely to be the address of an EOA.

Reply:
I decided not to, since in the worst case, you could use `withdrawTokens` again